components/script/script_thread.rs

```diff
@@ -3708,6 +3712,7 @@ impl ScriptThread {
             dummy_request_id,
             Ok(ResourceFetchTiming::new(ResourceTimingType::None)),
         );
+        context.append_parent_to_csp_list(policy_container.as_ref());
```

Comment:
It occurs to me that we should move this right after the process_response call, otherwise the srcdoc contents could initiate some blockable operations before we set up the CSP rules.

Reply:
Fixed 👍